src/test/groovy/test/TestUtils.groovy

```diff
@@ -19,16 +23,29 @@ final class TestUtils {
       .withPluginClasspath()
       .build()
   }
-
-  static def getLicenseText(def fileName) {
-    getClass().getResource("/license/${fileName}").text
+  static def myGetLicenseText(String fileName) {
```

Comment:
Can you keep this the same name?

Reply:
I have no objection to a spelling change (it is rather unimaginative!), but I'm not sure it's wise to end up with two functions with the same name in this case, because the scoping in groovy doesn't seem to be as clear as in Java. I had troubles getting this right (possibly my own fault) until I separated the spellings, and this strikes me as a case where future maintainers could easily find themselves confused.  If you insist..., but it was not an idle decision to change the name.

Reply:
Sure, they can be different names.

Reply:
Its ok to change the name but "my" getter is not a normal convention. Can we name this something more reasonable?